demos/smart_classroom_demo_gapi/main.cpp

```diff
@@ -546,6 +521,410 @@ bool ParseAndCheckCommandLine(int argc, char *argv[]) {
     return true;
 }
 
+} // namespace
+
+struct TrackerParamsPack {
+    TrackerParams tracker_reid_params;
+    TrackerParams tracker_action_params;
+};
+
+struct TrackersPack {    
+    TrackersPack( TrackerParams tracker_reid_params
+                , TrackerParams tracker_action_params) : tracker_reid(Tracker(tracker_reid_params))
+                                                       , tracker_action(Tracker(tracker_action_params)) {}
+    Tracker tracker_reid;
+    Tracker tracker_action;
+};
+
+namespace config {
+    detection::DetectorConfig face_config;
+    detection::DetectorConfig face_registration_det_config;
+    ActionDetectorConfig action_config;
+    FaceRecognizerConfig rec_config;
+} // namespace config
+namespace state {
+    /** NOTE: Questionable solution
+      * kernels use num_frames,
+      * num_frames total state need for FPS counter
+      */
+    size_t work_num_frames = 0;
+    size_t total_num_frames = 0;
+} // namespace state
+namespace util {
+    std::string GetBinPath(const std::string &pathXML) {
+        CV_Assert(pathXML.substr(pathXML.size() - 4, pathXML.size()) == ".xml");
+        std::string pathBIN(pathXML);
+        return pathBIN.replace(pathBIN.size() - 3, 3, "bin");
+    }
+    bool IsNetworkNewVersion(const std::string &model_path) {
+        CV_Assert(!model_path.empty());
+        return model_path.at(model_path.size() - 5) == '6';
+    }
+    struct Avg {
+        struct Elapsed {
+            explicit Elapsed(double ms) : ss(ms / 1000.),
+                mm(static_cast<int>(std::lround(ss / 60))) {}
+            const double ss;
+            const int    mm;
+        };
+
+        using MS = std::chrono::duration<double, std::ratio<1, 1000>>;
+        using TS = std::chrono::time_point<std::chrono::high_resolution_clock>;
+        TS started;
+
+        void    start() { started = now(); }
+        TS      now() const { return std::chrono::high_resolution_clock::now(); }
+        double  tick() const { return std::chrono::duration_cast<MS>(now() - started).count(); }
+        Elapsed elapsed() const { return Elapsed{ tick() }; }
+        double  fps(std::size_t n) const { return static_cast<double>(n) / (tick() / 1000.); }
+    };
+} // namespace util
+namespace cv
+{
+    namespace detail
+    {
+        template<> struct CompileArgTag<TrackerParamsPack>
+        {
+            static const char* tag()
+            {
+                return "custom.get_recognize_result_state_params";
+            }
+        };
+        template<> struct CompileArgTag<TrackerParams>
+        {
+            static const char* tag()
+            {
+                return "custom.get_action_detection_result_state_params";
+            }
+        };
+    }
+}
```

Comment:
Please use the consistent formatting across the code. In this case, `{` shouldn't be at a new line

Reply:
Fixed.